eng/pipelines/smoke-test.yml

```diff
@@ -105,4 +105,6 @@ jobs:
       - script: python ./common/smoketest/program.py
         displayName: "Run Smoke Test"
 
-      - template: ../common/TestResources/remove-test-resources.yml
```

Comment:
What about also updating the test template?

Reply:
Python does its own resource allocation for live tests. The smoke tests are the only places where we use dynamic resource deployment so we don't have to take a dependency on the preparer infrastructure.